drivers/remote/driver_test.go

```diff
@@ -177,6 +177,11 @@ func TestRemoteDriver(t *testing.T) {
 
 	var networkID string
 
+	handle(t, mux, "GetCapabilities", func(msg map[string]interface{}) interface{} {
+		return map[string]interface{}{
+			"Scope": "global",
+		}
+	})
```

Comment:
@WeiZhang555 My previous comment got overwritten due to the recent changes.

thanks for adding this test. in addition to this positive case, can you please add a couple of negative cases which am interested in seeing for backward compatibility ?

    empty return : return map[string]interface{}{ }
    return with extra params : return map[string]interface{}{ "Scope": "global", "foo":"bar" }